docs/antora-site-structure.md

```diff
@@ -0,0 +1,127 @@
+
+**Table of Contents**
+1. [Why Antora](#why-antora)
+2. [Scope of doc Repositories](#scope-of-doc-repositories)
+3. [Scope of Antora Definitions](#scope-of-antora-definitions)
+4. [Scope of Content Accessability](#scope-of-content-accessability)
+5. [Structure of Directories](#structure-of-directories)
+6. [Scope of Antora Definitions](#scope-of-antora-definitions)
+
+## Why Antora
+
+The main reasons for using [Antora][link-antora] is the following:
+
+1. It uses and extends the [asciidoc][link-asciidoc] text writing format
+
+2. It extends asciidoc with [multi-repo][link-playbook] capabilities. Doing so, the writer does not need to take care about repos anymore, as this is virtualized by Antora and content can be accessed with a standardized way
+
+3. Antora seperates writing the documentation and the [UI-Template][link-ui] to define how the content is presented
+
+## Scope of doc Repositories
+
+The ownCloud documentation is made of a master repo named `docs` which includes additional product repos like the clients or others (content sources, defined in `site.yml`). While each repo can be build individually for testing the outcome, only the build of docs creates a documentation which is pushed to the web.
+
+```
+layer-1     layer-2
+master      included repos
+
+            docs-client-desktop
+docs    --> docs-client-ios-app 
+            docs-client-android
+            ...
+
+```
+
+## Scope of Content Accessability
+
+Because Antora is capable defining additional ressources, you can access content from these resources where they are defined. To do so, you follow the [Resource ID Coordinates][link-resource-id] scheme.
+
+**Possible**
+```
+docs    --> docs-client-ios-app
+            └> index.adoc
+```
+**Impossible**
+```
+docs-client-ios-app    --> docs-client-desktop
+                           └> index.adoc
+```
+The reason why you cant access the above is, because `docs-client-ios-app` has no reference to `docs-client-desktop`. Only `docs` hast this info by the definition in its playbook `site.yml`.
+
+
+
+## Structure of Directories
+
+Each directory structure is defined identical in all doc repositories. This helps maintaining and accessing content easily.
+
+### The Antora Directory Stucture
+
+```
+modules/named-module-1/attachment
+                      /examples
+                      /images
+                      /pages
+                      /partials
+
+       /named-module-2/attachments
+                      /...
+       /...
+```
+
+Please see [Standard File and Directory Set][link-standard-directories] at `Example 1` for details.
+
+**Note that there is one important exception**:
+The navigation file `nav.adoc` is under the `partials` directory and not at the level of the named-module (like ROOT or admin ect). This is necessary because only files which are in a `family directory` can be accessed from the outside. This means, `docs` can access e.g. `nav.adoc` at any content source like `docs-client-ios-app`.
```

Comment:
```suggestion
The navigation file `nav.adoc` is under the `partials` directory and not at the level of the named-module (like ROOT or admin ect). This is necessary because only files which are in a `family directory` can be accessed from outside. This means `docs` can access e.g. `nav.adoc` at any content source like `docs-client-ios-app`.
```